packages/as-sha256/assembly/index.ts

```diff
@@ -141,6 +141,7 @@ function SIG1(x: u32): u32 {
   return rotr(x, 17) ^ rotr(x, 19) ^ (x >>> 10);
 }
 
+var tmpW: u32;
```

Comment:
Nitpick: prefer `let` ?

Reply:
yeah good catch, will apply for other places as well